src/templates/not_found/index.jsx

```diff
@@ -1,8 +1,14 @@
 import React from 'react';
+import Layout from '../../components/Layout';
 
 const NotFound = () => {
   return (
-    <h1>Page not found.</h1>
+    <Layout title="Dataset Publishers">
```

Comment:
Nitpick but this layout title should probably be changed

Reply:
I just pushed an update to change to `Page not found`